include/clang/AST/Expr.h

```diff
@@ -4505,8 +4505,8 @@ class GenericSelectionExpr : public Expr {
 
 /// \brief Represents a Checked C bounds expression.
 class BoundsExpr : public Expr {
-private:
```

Comment:
Sure, I'll put it back.  I noticed some of the other code in the file was taking advantage of the default behavior.